guides/common/modules/proc_deleting-multiple-content-view-versions.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Deleting_Multiple_Content_View_Versions_{context}"]
+= Deleting Multiple Content View Versions
+
+You can delete multiple content view versions simultaneously.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Content View*.
+. Click the content view you want to delete versions of.
+. Navigate to the *Versions* tab.
+. Select the check box of the version or versions you want to delete. 
+. Click the three dots icon at the top of the list of content views.
+. Click *Delete*.
+. Clicking delete opens the deletion wizard.
+. The deletion wizard displays any affected environments.
+. If there are no affected environments, review the details and click *Delete*. 
+. If there are any affected environments, the deletion wizard has the option to reassign any hosts or activation keys before deletion.
```

Comment:
I'd make this into substeps of the previous step, itemized list

Reply:
PLease no substeps :)

Reply:
These are not steps all the same, especially line 16.

Reply:
@melcorr @Lennonka should I leave these two lines as they are then?